lib/core/src/lib/form/components/widgets/base-viewer/base-viewer.widget.ts

```diff
@@ -44,12 +44,13 @@ export class BaseViewerWidgetComponent extends WidgetComponent implements OnInit
     }
 
     ngOnInit(): void {
-        if (this.field &&
-            this.field.value &&
-            Array.isArray(this.field.value) &&
-            this.field.value.length) {
-            const file = this.field.value[0];
-            this.field.value = file.id;
+        if (this.field && this.field.value) {
+            if (Array.isArray(this.field.value) && this.field.value.length) {
```

Comment:
these statements also seem to check the same thing

Reply:
This statment is checking whether the field value is an array and if it contains at least one element

Reply:
Wouldn't the `length` property exist even if there are no elements in the array? If the length is 0 then `if (this.field.value.length)` still returns true?

Reply:
no `if(0)` always return false, but I can do it more explicit using the greater than operator